src/App.tsx

```diff
@@ -1,61 +1,112 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { preperedTodos } from './services/preparedTodos';
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [isUserError, setIsUserError] = useState(false);
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [todos, setTodos] = useState(todosFromServer);
+  const preparedTodos: Todo[] = preperedTodos(todos);
+
+  const resetFormFileds = () => {
+    setTitle('');
+    setUserId(0);
+  };
+
+  const getTodo = () => {
```

Comment:
```suggestion
  const createTodo = () => {
```
think best named create or generate
Also it`s func can be helper outside component file ( for future )